src/tracing/server.coffee

```diff
@@ -0,0 +1,25 @@
+###
+This server response to ui on a /vizc end point and returns queried JSON as reponse
+###
+express = require 'express'
+bodyparser = require 'body-parser'
+varz = require 'express-varz'
+{conf} = require 'rainier/conf'
+port = conf.get 'vizc_server_port'
+vizJsonCreator = require './vizceral_json_file_creator_nodes'
+
+server = express()
+server.use bodyparser.json limit: '10mb'
```

Comment:
This is for post bodies. That is people sending us data in requests, like uploading files to the server.